src/cbc_sdk/platform/alerts.py

```diff
@@ -1106,6 +1311,24 @@ def _perform_query(self, from_row=1, max_rows=-1):
                 still_querying = False
                 break
 
+    def results_compatibility_mapping(self, results):
+        v6_query = type(self)
```

Comment:
I'm confused how is this defined as v6 query?

Reply:
removed. this is when i was before I was on the correct path